filters/builtin/compress.go

```diff
@@ -99,7 +99,7 @@ func (e encodings) Swap(i, j int) { e[i], e[j] = e[j], e[i] }
 //
 // Example:
 //
-// 	* -> compress() -> "https://www.example.org"
+//   - -> compress() -> "https://www.example.org"
```

Comment:
These changes look wrong, probably because example eskip is not a proper code block so godoc confuses it with item list